src/app/tests.py

```diff
@@ -1119,10 +1120,14 @@ def test_archive_license_requests_feature(self):
         license_name = driver.find_element_by_css_selector('td').text
         self.assertEquals(license_name, "BSD Zero Clause License-00")
         self.assertEquals(LicenseRequest.objects.get(id=license_obj.id).archive, False)
-        driver.find_element_by_id('archive_button' + str(license_obj.id)).click()
-        driver.find_element_by_id('confirm_archive').click()
-        self.assertEquals(LicenseRequest.objects.get(id=license_obj.id).archive, True)
+        if driver.find_element_by_id('archive_button' + str(license_obj.id)):
```

Comment:
When will the test reach here? I think if the user has access to github_login, only then he will be able to see the feature. I think we run the test for a simple user so code will never go into `if`. Can you check for that or is there a way to run the same test with different user? 
You can either create a new issue for that and work on that separately, I'll merge this. Let me know what you find.

Reply:
@anshuldutt21 Can you create one now, I'll merge this then. Just to be on the safe side of not forgetting about this one. Also, then anyone can pick up the task if you are busy.

Reply:
Done

Reply:
Thanks! Merging